plugins/reporters/freemarker/src/main/kotlin/FreemarkerTemplateProcessor.kt

```diff
@@ -224,13 +224,12 @@ class FreemarkerTemplateProcessor(
         }
 
         /**
-         * Return only those [licenses] that are classified under the given [category], or that are not categorized at
-         * all.
+         * Return only those [licenses] that are classified under the given [category].
          */
         @Suppress("UNUSED") // This function is used in the templates.
         fun filterForCategory(licenses: Collection<ResolvedLicense>, category: String): List<ResolvedLicense> =
             licenses.filter { resolvedLicense ->
-                input.licenseClassifications[resolvedLicense.license]?.contains(category) != false
+                input.licenseClassifications[resolvedLicense.license]?.contains(category) ?: false
```

Comment:
The notice templates still uses `filterForCategory()`, and IIUC these intends to include exactly the licenses which are either / or:

1. uncategorized
2. categorized with the (provided) `include-in-notice` category.

I suppose that this behavior shall stay as is. So, I think this PR should account for that.

Reply:
Not sure if I understand you correctly.
Is your proposal 
1. Create a new function that returns the uncategorized licenses in addition to the proposed change? And additionally adjust the notice template?
Or is it
2. Closing this PR and keeping the functionality as-is?

Reply:
> Not sure if I understand you correctly.

ORT has the files `NOTICE_DEFAULT.ftl` and `NOTICE_SUMMARY.ftl`.
I believe that the output should of these templates should remain unaffected by this PR. 
This could be achieved with option #1 above IIUC.